v0/sources/shopify/transform.js

```diff
@@ -27,16 +37,37 @@ const ecomPayloadBuilder = (event, shopifyTopic) => {
   message.setEventType(EventType.TRACK);
   message.setEventName(RUDDER_ECOM_MAP[shopifyTopic]);
 
-  const properties = createPropertiesForEcomEvent(event);
+  let properties = createPropertiesForEcomEvent(event);
```

Comment:
Probably an out-of-scope question, but why should there be any difference in the `trackPayloadBuilder` and `ecomPayloadBuilder` methods? Except for the event name, seems like the other differences can be made even across both of them.

Reply:
Only diff is `let properties = createPropertiesForEcomEvent(event);` we can definitely do that for now I think we should go ahead with the items that are tested and on boarded because of timelines